osf-cas/templates/configmap.yaml

```diff
@@ -832,7 +893,7 @@ services/{{ $key }}-{{ $val.id }}.json: |-
     "description": "",
     "id": {{ $val.id }},
     "evaluationOrder": 10,
-    "logo": "",
+    "logo": "https://{{ $casDomain }}/images/branded/preprints-{{ $key }}-logo.png",
```

Comment:
This was removed for the initial staging3 deployment due to the logo images not available at that time.

In addition, moved all branded logos / shields into their own folder / path.